interpreter/meta/travis/install-ocaml.sh

```diff
@@ -12,16 +12,29 @@ cd $(dirname ${BASH_SOURCE[0]})/../../..
 rm -rf ocaml
 mkdir ocaml
 cd ocaml
-curl https://wasm.storage.googleapis.com/ocaml-4.02.2.tar.gz -O
-CHECKSUM=$(shasum -a 256 ocaml-4.02.2.tar.gz | awk '{ print $1 }')
+curl https://wasm.storage.googleapis.com/ocaml-4.05.0.tar.gz -O
```

Comment:
It seems odd to have one tarball in google storage and the other from github. We might as well just put them both in google storage, or switch this to https://github.com/ocaml/ocaml/archive/4.05.0.tar.gz ?